sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -659,6 +659,14 @@ public partial class RetryOptions
         public System.TimeSpan NetworkTimeout { get { throw null; } set { } }
     }
     public delegate System.Threading.Tasks.Task SyncAsyncEventHandler<T>(T e) where T : Azure.SyncAsyncEventArgs;
+    public partial class TelemetryDetails
+    {
+        public TelemetryDetails(System.Reflection.Assembly assembly, string? applicationId = null) { }
+        public string? ApplicationId { get { throw null; } }
+        public System.Reflection.Assembly Assembly { get { throw null; } }
+        public string UserAgentString { get { throw null; } }
```

Comment:
Also seems a bit strange to suffix this with "String", as opposed to simply UserAgent.

Reply:
> Shouldn't we still override ToString?

If this type ever does more than represent the UserAgent string, then ToString's implementation would need to change. For this reason, we should probably preemptively do this to avoid a breaking behavior change in the future.

Reply:
ToString implementations can evolve to contain more information, right?

Reply:
If we are not going to implement it, should we still override and call base implementation, and hide it from intellisense?